### DISCUSSION TITLE
pacemaker details not populating on dashboard

### DISCUSSION THREAD

**Discussion by one-kash** [ClusterLabs/ha_cluster_exporter#202] [Q&A] [answered]
Hello Folks,

I am trying to use the Grafana dashboard as [documented here. ](https://github.com/ClusterLabs/ha_cluster_exporter/blob/main/dashboards/README.md) but Pacemaker and Corosync section is not populating (screenshot below) at all.. any ideas what I am missing?

more details of the config below:

- have [ha_cluster_exporter](https://github.com/ClusterLabs/ha_cluster_exporter) configured
- have [node_exporter](https://github.com/prometheus/node_exporter) configured
- enabled these collectors with node exporter: --collector.systemd --collector.drbd --collector.perf 
- Prometheus yml configuration below
- IP: 9664/metrics shows the pacemaker data

global:
  scrape_interval: 15s

scrape_configs:
  - job_name: ABC-CS
    static_configs:
    - targets: ['sapascs:9100','sapers:9100']

remote_write:
  - url: "url"
    basic_auth:
      username: " <name>"
      password:  "<pass>"

<img width="914" alt="image" src="https://user-images.githubusercontent.com/26795040/192617163-3b2af134-64c9-4df5-98a1-82106975731f.png">

**Comment by one-kash** [comment, marked as answer]
after I typed everything, I realized that ha_cluster has different port from node_exporter and that (IP: 9664) needed to be added in the config file as targets. I guess reading the details works better than just running it in my head!

**Comment by diegoakechi** [comment]
That was easy @ravkash! thanks for documenting it here ;)

**Reply by one-kash** [reply]
yep, I like easy stuff! :)
